contract/propertyMarketplace.sol

```diff
@@ -84,192 +135,170 @@ contract PropertyMarketplace {
             _houseToken
         );
         propertiesLength++;
-        
-
     }
 
-    function readProperty(uint _index) public view returns (
-        address payable,
-        PropertyLabel memory,
-        PropertyStockData memory,
-        uint, 
-        uint, 
-        Status,
-        HouseToken
-    ) {
-        return (
-            properties[_index].owner,
-            properties[_index].label, 
-            properties[_index].stockData,
-            properties[_index].bedrooms,
-            properties[_index].bathrooms,
-            properties[_index].status,
-            properties[_index].houseToken
-        );
+    /// @dev returns the data of a property
+    function readProperty(uint _index)
+        public
+        view
+        exist(_index)
+        returns (Property memory)
+    {
+        return (properties[_index]);
     }
 
-    function getPropertiesLength()
-    public
-    view
-    returns (uint){
+    function getPropertiesLength() public view returns (uint) {
```

Comment:
there are some times you change the function signatures to have one item per line and then in this case it's been compressed to one line. What's the reason?

Reply:
It's my formatting extension (I believe it's currently set on the hardhat + solidity)